extension/src/popup/components/ErrorBoundary/index.tsx

```diff
@@ -54,7 +54,7 @@ export const UnhandledError = ({
   // expected to work outside of <Router />
   const isOnAccount = window.location.hash === "#/account";
   return (
-    <View>
+    <React.Fragment>
```

Comment:
does removing `View` here and in subsequent views still make full screen mode work?

Reply:
yeah it does. The <View> is still above these components in the Outlet now, we have to remove these because having 2 Views nested created weird layout bugs but the rendered dom should be very similar in hierarchy.